.github/workflows.yml

```
@@ -0,0 +1,20 @@
+     name: Check PR sync with main
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 5

(indentation)